pyproject.toml

```diff
@@ -22,12 +22,14 @@ dependencies = ["numba>=0.59.1"]
 
 [project.optional-dependencies]
 cu11 = [
+    "cuda-core[cu11]",
```

Comment:
We should pin these to a specific version for the time being. We don't have any API guarantees in `cuda-core` yet and it's very likely we will break things out from underneath you.

Reply:
It should have been `cuda-bindings`, sorry. Linker PR changes creeping in.